libs/application-generic/src/services/analytic-logs/log.repository.ts

```diff
@@ -208,15 +213,22 @@ export abstract class LogRepository<TSchema extends ClickhouseSchema<any>, TEnha
       }
 
       // Handle structured conditions {field, operator, value}
-      if (!('field' in condition) || !('operator' in condition) || !('value' in condition)) {
-        throw new Error('Each condition must have field, operator, and value properties');
+      if (!('field' in condition) || !('operator' in condition)) {
+        throw new Error('Each condition must have field and operator properties');
       }
 
       const { field, operator, value } = condition;
       this.validateColumnName(field as SchemaKeys<TSchema>);
       this.validateOperator(operator);
 
-      if (value === null || value === undefined) {
+      // NULL operators don't need values
+      const nullOperators: NullOperators[] = ['IS NULL', 'IS NOT NULL'];
```

Comment:
@djabarovgeorge please take a look on this improvement, but seems working ;)

Reply:
also I noticed that the logic here is messed up a bit, because in line 236 we always build conditions with the `AND` clause which is ok for the activity feed, but might not be ok, if we would need to build the `OR` conditions;
what do I mean:
```
condition1 AND condition2 AND condition3...
```
but we won't be able to do:
```
condirion1 OR condition2 OR condition3...
```
because the conditions with OR will be always joined with AND:
```
(condition1 OR condition2) AND (condition3 OR condition4)
```

Reply:
totally agree. this module has grown with the project needs, and we should have some boundaries around how much support we want to add here. the goal should be to keep it as simple as possible to maintain for common use cases. for more complex use cases, we will always have the option to use native sql queries, just like we do on the analytics page.